src/Target.h

```diff
@@ -61,6 +61,8 @@ struct Target {
 
         RegisterMetadata,  ///< Generated code registers metadata for use with halide_enumerate_registered_filters
 
+        Matlab,  ///< Enable Matlab support code.
```

Comment:
More detail in this comment please (e.g. Also compile in a matlab wrapper function so that the generated code can be used with mex).